modules/querying/pages/func/query-user-defined-functions.adoc

```diff
@@ -51,13 +54,10 @@ inline string reverse(string str){
 
 [CAUTION]
 ====
-If any code in `ExprFunctions.hpp` or `ExprUtil.hpp` causes a compilation error, GSQL cannot install any GSQL query, even if the GSQL query doesn't call any query UDF. Therefore, please test each new query UDF after adding it. One way of testing a function is to create a new file `test.cpp` and compile it: +
-`> g{pp} test.cpp
+If any code in `ExprFunctions.hpp` or `ExprUtil.hpp` causes a compilation error, GSQL will be unable to install _any_ new queries, whether user-defined or not. Therefore, please test each new query UDF after adding it. One way of testing a function is to make a new file `test.cpp` and compile it.
 
-____
-./a.out``
-You might need to remove the include header ``#include <gle/engine/cpplib/headers.hpp>`` in ``ExprFunctions.hpp`` and ``ExprUtil.hpp` in order to compile.
-____
+You might need to remove the included header `#include <gle/engine/cpplib/headers.hpp>` in `ExprFunctions.hpp` and `ExprUtil.hpp` in order to compile.
+====
```

Comment:
I think this `#include` header is what allows users to use accumulators. In the current version, it no longer exists in the UDF file you download. When you upload it, the include directive is applied to your UDF code automatically.